source/agora/consensus/state/Ledger.d

```diff
@@ -916,13 +912,18 @@ public class Ledger
     /// Ditto
     public bool hasMajoritySignature (Height height) @safe nothrow
     {
+        ValidatorInfo[] validators;
+        try
+            validators = this.getValidators(height);
+        catch (Exception exc)
+            return false;
```

Comment:
Even better:
https://github.com/bosagora/agora/blob/a104402593c183ecae84221989ceba4d0fe62e31/source/agora/consensus/state/Ledger.d#L258-L281

Reply:
But I think we need the `validators` in the function we call as when we use the `overload` we need the `utxo` from `VailidatorInfo`. If we only `getValidators` in the `overload` then we will sometimes get them twice.